internal/data/spicedb.go

```diff
@@ -35,6 +35,12 @@ const (
 	relationPrefix = "t_"
 )
 
+var (
```

Comment:
Have you considered making `consistency` a variable in the SpiceDbRepository struct rather than a global variable? Feels like this would be the right scope.

Reply:
That would be a nice place to put it, thanks!